### PR TITLE
completed task

### PR DESCRIPTION
**Pull request by TKuts**
- [DEMO LINK](https://TKuts.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://TKuts.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -2,15 +2,51 @@
...
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="get"
+      class="search__bar search__bar--big"
+      data-qa="big">
```
Comment (polosanya) [inline src/index.html:17]:
```suggestion
    <form
      action="#"
      method="get"
      class="search__bar search__bar--big"
      data-qa="big"
    >
```
Fix all cases
image

File: src/index.html [hunk 2]
```diff
@@ -2,15 +2,51 @@
...
+        <input
+          type="text"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+          class="search__bar-input search__bar-input--big">
+      </label>
+    </form>
+
+
+    <form
```
Comment (polosanya) [inline src/index.html:32]:
```suggestion
    </form>

    <form
```

File: src/index.html [hunk 3]
```diff
@@ -2,15 +2,51 @@
...
+          type="text"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+          class="search__bar-input search__bar-input--small">
+      </label>
+    </form>
+
   </body>
+
 </html>
```
Comment (polosanya) [inline src/index.html:52]:
```suggestion
      </label>
    </form>
  </body>
</html>
```
Don't leave empty lines between parent and child elements

**Review by polosanya — CHANGES_REQUESTED** [review]
Please don't forget to add demo and test links to the PR description, so we can properly check your work
image

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  color: #3d4e61;
+  font-weight: 400;
```
Comment (IvanFesenko) [inline src/style.css:20]: `font-weight` by default is redudant

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+.search__bar--big {
+  height: 70px;
+}
+
+.search__bar--small {
+  height: 42px;
+}
```
Comment (IvanFesenko) [inline src/style.css:41]: Considuer use padding for input element instead of hardcodding height

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+
+.search__bar-input--big {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+.search__bar-input--small {
+  font-family: "Avenir", sans-serif;
+  font-size: 14px;
+}
```
Comment (IvanFesenko) [inline src/style.css:73]: font-family is already setted for html, for input use inherit value. Also do not repeat ypur self
```suggestion
.search__bar-input {
  width: 100%;
  border-radius: 4px;
  border: none;
  font-family: inherit;
}

.search__bar-input--big {
  font-size: 16px;
}

.search__bar-input--small {
  font-size: 14px;
}
```

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+
+.search__bar:hover {
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+}
+
+.search__bar-input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  font-family: Avenir, sans-serif;
```
Comment (IvanFesenko) [inline src/style.css:81]:
```suggestion

```

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
Good job. Some more improvements required. Check my comments

File: src/style.css [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+.search__bar {
+  margin-top: 20px;
+  display: flex;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+.search__bar--big {
+  padding: 22px 0;
+}
```
Comment (sTorba24) [inline src/style.css:36]: move it to search__bar just

Comment (sTorba24) [reply src/style.css:36]: image

File: src/style.css [hunk 9]
```diff
@@ -0,0 +1,87 @@
...
+.search__bar-input {
+  width: 100%;
+  border-radius: 4px;
+  border: none;
+  font-family: inherit;
+}
+
+.search__bar-input--big {
+  font-size: 16px;
+}
```
Comment (sTorba24) [inline src/style.css:67]: same, move it to search__bar-input

**Review by sTorba24 — CHANGES_REQUESTED** [review]
Almost done! Small fixes to make it perfect

**Review by polosanya — CHANGES_REQUESTED** [review]
One small but important fix left 🙏 
image
Almost, pay attention to the background on hover
image

**Comment by TKuts** [comment]
image

I'm sorry, but I don't understand thus mistake. You can see that Figma's style is changing. How do you know what shouldn't change?

**Review by polosanya — CHANGES_REQUESTED** [review]
The text that user enters inside the input should change on focus, but the placeholder shouldn't.
image
If you have any questions, please ask them in fe_chat and tag mentors. It will be faster because your answers in GitHub will be visible to the mentor only after you re-request review 🙂

**Pull request closed without merging**